xrfeitoria/utils/runner.py

```diff
@@ -117,10 +117,28 @@ def __init__(
         if not self.engine_exec.exists() or not self.engine_exec.is_file():
             raise FileExistsError(f'Engine executable not valid: {self.engine_exec}')
         logger.info(f'Using engine executable: "{self.engine_exec.as_posix()}"')
-        self.project_path = Path(project_path).resolve() if project_path else None
+        if project_path:
+            self.project_path = Path(project_path).resolve()
+            assert self.project_path.exists(), (
+                f'Project path not valid: "{self.project_path.as_posix()}"\n'
+                'Please check in `xf.init_blender(project_path=...)` or `xf.init_unreal(project_path=...)`'
```

Comment:
A few typos:
- "Project path `is` not valid" or "Project path does not exist"
- Please check `xxx`

Reply:
resolved

Reply:
fixed